partner_age/models/partner.py

```diff
@@ -27,18 +34,15 @@ def _search_age(self, operator, value):
     @api.multi
     @api.depends("birthdate_date")
     def _compute_age(self):
-        self.ensure_one()
-        if self.birthdate_date:
-            dBday = datetime.strptime(
-                str(self.birthdate_date), OE_DFORMAT
-            ).date()
-            dToday = datetime.now().date()
-            self.age = (
-                dToday.year
-                - dBday.year
-                - ((dToday.month, dToday.day) < (dBday.month, dBday.day))
-            )
-
-    age = fields.Integer(
-        string="Age", compute="_compute_age", search="_search_age"
-    )
+        for partner in self:
+            if partner.birthdate_date:
+                birthday = partner.birthdate_date
+                today = datetime.now().date()
+                partner.age = (
+                    today.year
+                    - birthday.year
+                    - (
+                        (today.month, today.day)
+                        < (birthday.month, birthday.day)
+                    )
+                )
```

Comment:
oui si tu remplaces un api.one par un multi faut se poser la question de l'usage de l'ensure_one qui ne devrait se mettre que lorsque la fonction est appelée sur un bouton.

Donc oui une itération sur le self est plus adaptée d'autant plus lorsque ce champ est affiché dans une liste :p